server/tests/articleModels.test.js

```diff
@@ -1,30 +1,33 @@
-import faker from 'faker';
 import { expect } from 'chai';
 import db from '../models';
 
 
 const validArticle = {
-  title: faker.random.word(),
-  body: faker.random.words(),
```

Comment:
Faker library is causing irregularity on the test case, so I decided to replace it with plain text.